drop-storage/src/lib.rs

```diff
@@ -32,4 +35,671 @@ impl Storage {
             conn,
         })
     }
+
+    pub async fn insert_transfer(
```

Comment:
The queries will be executed one by one and will cause a race behavior. For example, the paths might not be yet added completely so the API might return an incomplete state.

To fight this databases have this concept - transaction. https://docs.rs/sqlx/latest/sqlx/struct.Transaction.html sqlx seems to support it. So basically all queries that should be atomic, should be wrapped in a transaction.

We might not encounter an issue as we lock the drop-storage instance in a mutex and only expect our own app to read it. Still leaving the comment just to be aware

Reply:
done